Makefile

```diff
@@ -37,6 +37,9 @@ package:
 	tar -C build/Build/Products/Release -cvJf build/Build/Products/Release/zld.$(HASH).tar.xz zld
 	tar -C build/Build/Products/Release -cvJf build/Build/Products/Release/zld.dSYM.$(HASH).tar.xz zld.dSYM
 
+github_release: build
+	tar -C build/Build/Products/Release -cvJf build/Build/Products/Release/zld.tar.xz zld
```

Comment:
we've historically used .zip and not .tar.xz, any issue with .zip?

Reply:
No issue. `make package` uses `.tar.xz`, I just copy that.

Reply:
Actually `zld.tar.xz` is about half the size of `zld.zip`, so any reason not using that?

Reply:
tar is fine